project_task_pull_request/models/project_project.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2017 Specialty Medical Drugstore
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class Project(models.Model):
+    _inherit = "project.project"
+    _name = "project.project"
```

Comment:
When `_name` = `_inherit`, it is not necessary to set:

 ```suggestion
    _inherit = "project.project"
```